### PR TITLE
[Konvoy] KIB Override page updates

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-87545

## Description of changes being made

Un-linked override files from main and pasted content from directory to the docs site as requested by support.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4495.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md [hunk 1]
```diff
@@ -10,9 +10,49 @@ menuWeight: 0
...
+
+- Offline override:
+
+  ```yaml  
+  os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_major_version }}_x86_64.tar.gz"
+  pip_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/pip-packages.tar.gz"
+  images_local_bundle_dir: "{{ playbook_dir}}/../artifacts/images"
+  ```
+
+- Offline for FIPS override:
```
Comment (some-things) [inline pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md:51]: In the preview, this section appears blank: http://docs-d2iq-com-pr-4495.s3-website-us-west-2.amazonaws.com/dkp/konvoy/2.2/image-builder/override-files/default-or-files/

File: pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md [hunk 2]
```diff
@@ -10,9 +10,49 @@ menuWeight: 0
...
+
+  ```yaml
+# fips os-packages
```
Comment (some-things) [inline pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md:54]: Formatting broken here in preview.

File: pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md [hunk 3]
```diff
@@ -10,9 +10,49 @@ menuWeight: 0
...
-- Offline override file.
-- Offline for FIPS override file.
+- FIPS override:
+  ```yaml
+  ---
+  k8s_image_registry: docker.io/mesosphere
 
-You can find these override files in the [Konvoy Image Builder repo](https://github.com/mesosphere/konvoy-image-builder/tree/main/overrides).
+  fips:
+  enabled: true
```
Comment (some-things) [inline pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md:19]: Syntax broken here as opposed to the live files: https://github.com/mesosphere/konvoy-image-builder/blob/main/overrides/fips.yaml

File: pages/dkp/konvoy/2.0/image-builder/override-files/index.md [hunk 4]
```diff
@@ -9,4 +9,36 @@ menuWeight: 75
 ---
 ## Override files
```
Comment (some-things) [inline pages/dkp/konvoy/2.0/image-builder/override-files/index.md:10]: My main concern initially was maintainability. For example, if something changes or is added in a new version of KIB that isn't in an older version, the overrides would not be valid for that version. These changes are definitely more 'static', but docs/engineering will need to make sure that they are maintained and updated according to any changes to KIB.

**Review by some-things — CHANGES_REQUESTED** [review]

**Comment by msdolbey** [comment]
@some-things @mwrightd2 - 
I took a look at this, and I updated the yaml (fixed that one formatting thing, too) - and updated the yaml to go off of what version of KIB corresponds with what version of DKP
DKP 2.0 = 1.0
DKP 2.1 = 1.5.0
DKP 2.2 = 1.10.0

If you need changes you'll have to pull first - but after this - it looks solid to me.

**Review by msdolbey — APPROVED** [review]
👍 
OVERRIDES

**Pull request merged**